cyflash/bootload.py

```diff
@@ -206,6 +277,17 @@ def progress(self, message=None, current=None, total=None):
 
 def main():
 	args = parser.parse_args()
+
+	if (args.logging_config):
+		import sys
```

Comment:
Removed sys. Logging needs to be initialized before python-can if a logging configuration file has been specified on the command line.